packages/typespec-azure-core/test/models.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expectDiagnosticEmpty } from "@typespec/compiler/testing";
+import { getHttpPart, isOrExtendsHttpFile } from "@typespec/http";
+import { ok, strictEqual } from "assert";
+import { describe, it } from "vitest";
+import { getOperations } from "./test-host.js";
+
+describe("typespec-azure-core: models", () => {
```

Comment:
remove top level describe

Reply:
I am not familiar with the test framework but this case calls async function `getOperations` which can not be used directly in `describe`.

Reply:
Just this line and the closing }) keep the content at the root. That was added when we were using mocha and all tests are merged together so needed some way to have a unique top level. But vitest already scope per file

Reply:
I see.
Updated.